server/midddleware/metrics.go

```diff
@@ -19,13 +19,12 @@ import (
 	"errors"
 	"strconv"
 
-	"github.com/tigrisdata/tigris/server/request"
-	ulog "github.com/tigrisdata/tigris/util/log"
-	"github.com/uber-go/tally"
-
 	"github.com/apple/foundationdb/bindings/go/src/fdb"
 	api "github.com/tigrisdata/tigris/api/server/v1"
 	"github.com/tigrisdata/tigris/server/metrics"
+	"github.com/tigrisdata/tigris/server/request"
```

Comment:
Nice, whitespace commando at its best!!! :- )

Reply:
This was goland I think :).